src/js_parser.zig

```diff
@@ -24136,10 +24086,9 @@ pub fn newLazyExportAST(
     return result.ast;
 }
 
-const CommonJSWrapper = union(enum) {
-    none: void,
-    bun_dev: Expr,
-    bun_js: void,
+const WrapMode = union(enum) {
```

Comment:
should this just be an enum?

Reply:
yea this can now be an enum since bun_dev was deleted